README.md

```diff
@@ -6,43 +6,43 @@ Create and verify JSON Web Tokens with deno.
 
 ### create
 
-Takes a `header`, `payload` and `key` and returns the url-safe encoded `jwt`.
+Takes `header`, `payload` and `key` and returns the url-safe encoded `jwt`.
 
 ```typescript
-import { create } from "https://deno.land/x/djwt@$VERSION/mod.ts"
-
-const jwt = await create({ alg: "HS512", typ: "JWT" }, { foo: "bar" }, "secret")
-// eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJmb28iOiJiYXIifQ.WePl7achkd0oGNB8XRF_LJwxlyiPZqpdNgdKpDboAjSTsWq-aOGNynTp8TOv8KjonFym8vwFwppXOLoLXbkIaQ
+import { create } from "https://deno.land/x/djwt@$VERSION/mod.ts";
+const key = await crypto.subtle.generateKey(
```

Comment:
`importKey` method should also be added to README.

Reply:
Hi thanks for taking the time. I will add `importKey` but `generateKey` has already been implemented and it works: https://github.com/denoland/deno/blob/main/ext/crypto/00_crypto.js

The reason I am wary about using `importKey` is that people might use too short or not recommended key lengths.

Reply:
Sorry, I was wrong just now. deno has not yet implemented `exportKey`.